tiledb/api/c_api/vfs/test/unit_capi_vfs.cc

```diff
@@ -331,6 +333,10 @@ TEST_CASE("C API: tiledb_vfs_move_dir argument validation", "[capi][vfs]") {
 }
 
 TEST_CASE("C API: tiledb_vfs_copy_dir argument validation", "[capi][vfs]") {
+  if constexpr (tiledb::platform::is_os_windows) {
+    // This API is not yet supported on Windows.
```

Comment:
I don't see any follow-up shortcut story about this - @KiterLuc @teo-tsirpanis please make sure to file follow-ups for missing features (or else cross-link if it already exists and I missed it)

Reply:
https://app.shortcut.com/tiledb-inc/story/34109/vfs-copy-file-copy-dir-for-windows